lib/tasks/mailer.rake

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+require 'yaml'
+
+namespace 'mailer' do
+  task start: :environment do
+    config_path = ENV.fetch('MAILER_CONFIG_PATH', 'config/mailer.yml')
+    config = YAML.load_file(config_path).deep_symbolize_keys
+
+    Rails.logger = ActiveSupport::Logger.new(STDOUT, level: :debug)
+
+    workers = []
+    config[:events].each do |event|
+      exchange = config[:exchanges][event[:exchange].to_sym]
+      keychain = config[:keychain][exchange[:signer].to_sym]
+
```

Comment:
why we would use so many workers?
Please only 1 process for 1 queue

Reply:
Fixed in next commits. You were right, we can bind a lot exchanges into one queue.